server/constants/tables.json

```diff
@@ -1,5 +1,9 @@
 {
     "appointment": "Appointment",
     "student": "Student", 
-    "worker": "Service_Worker"
+    "worker": "Service_Worker", 
+    "timeslot": "Timeslot", 
+    "worker_timeslot": "Worker_Timeslot",
```

Comment:
Hi Peter,
Small change, but `_` is normally used for database variables and lowerCamelCase is used for the backend variables. Thus, I would suggest renaming `worker_timeslot` to `workerTimeslot` and same for `school_worker`.

Reply:
Resolved! Thanks for the good eye!